### ISSUE TITLE
[Feature] "Copy to device..." should include option to also copy present album cover.

### ISSUE THREAD

**Issue by 3ddysan** [jonaski/strawberry#31]

**Comment by jonaski** [comment]
I'll definitely give this a try. But I'm planning a new release first.

**Comment by jonaski** [comment]
I'm working on copying album cover to libgpod compatible devices.
However I have some problems testing it, I got 2 iPods, one, which I think is a 5th generation 60GB had rockbox on it, but I've reset it to factory defaults, it copies the album art successfully, but it won't show up. I'm not entirely sure album art is supported on it.
I also got a newer iPod, which I think is a 6th generation classic, it won't mount on linux at all, just gets disconnected. I tested it on a windows PC and there it works, I think it might be related to newer kernels.
So it would be great if someone could test it.
